src/args.c

```diff
@@ -759,9 +759,10 @@ void parse_args(int argc, char* argv[])
 			printf("\t%s--list-dhcp6%s        List known DHCPv6 config options\n\n", green, normal);
 
 			printf("%sDebugging and special use:%s\n", yellow, normal);
-			printf("\t%sd%s, %sdebug%s            Enter debugging mode\n", green, normal, green, normal);
-			printf("\t%stest%s                Don't start pihole-FTL but instead\n", green, normal);
-			printf("\t                    quit immediately\n");
+			printf("\t%sd%s, %sdebug%s            Enter debugging mode:\n", green, normal, green, normal);
+			printf("\t							  Don't go into daemon mode + verbose logging");
```

Comment:
![Screenshot from 2023-06-10 13-11-51](https://github.com/pi-hole/FTL/assets/16748619/c3a70995-6b29-4e9e-8e1d-8b58131c4eed)

Reply:
There is are several line breaks missing here and the second sentence is way too much indented.

Reply:
![Screenshot at 2023-08-01 22-18-00](https://github.com/pi-hole/FTL/assets/26622301/ba20c362-79c4-473c-9918-bea6d44d2db5)